build/cmake/toolkit.cmake

```diff
@@ -126,14 +126,32 @@ if(UNIX AND NOT APPLE AND NOT WIN32 AND (WXX11 OR WXGTK2 OR (WXGTK AND wxHAVE_GD
 endif()
 
 if(WXQT)
-    set(QT_COMPONENTS Core Widgets Gui OpenGL Test)
+    set(QT_COMPONENTS Core Widgets Gui OpenGL OpenGLWidgets Test)
+
+    find_package(Qt6 COMPONENTS ${QT_COMPONENTS})
+    if (Qt6_FOUND)
+        set(QT_MAJOR_VERSION 6)
+
+        find_package(PkgConfig REQUIRED)
+
+        pkg_check_modules (GLIB2 REQUIRED glib-2.0)
+        include_directories(${GLIB2_INCLUDE_DIRS})
```

Comment:
This should be removed. We definitely do not wan't to add them to each target. Is adding them to `wxTOOLKIT_INCLUDE_DIRS` below not enough?

Reply:
Seems enough, now fixed.